licenseupdater/README.md

```diff
@@ -0,0 +1 @@
+# License Updater
```

Comment:
Maybe it would be good to add one paragraph how to use it. 

1. How to build?
2. How run it?
3. What flags are available?